changelog.d/9068.feature

```diff
@@ -0,0 +1 @@
+Add experimental support for handling `/keys/claim` and `/room_keys` APIs on worker processes.
```

Comment:
Should this be combined with the one from #9044?

Reply:
Err, don't know really. I don't know whether its better to have the separate or combine them (in terms of readability).

(I'm going to leave it for now to let the tests run)